### DISCUSSION TITLE
Should use double quote instead of single quote with tsx-ts-mode

### DISCUSSION THREAD

**Discussion by gormv** [joaotavora/eglot#1359] [General]
I'm a bit puzzled with this issue and wonder if anyone has an idea on how to fix my issue or if it's considered a bug? Or where this is actually failing or if it's possible to configure somehow.

The issue is that when I use completion-at-point in a TSX file using eglot with tsx-ts-mode, it will expand with single quote instead of double quote.

To reproduce:

1. Run with this init.el file

```
;; seems to be required to enable <div classNa[M-x completion-at-point]
(use-package yasnippet
  :ensure t
  )

(use-package eglot
  :custom
  (eglot-send-changes-idle-time 0.1)
  (eglot-extend-to-xref t))

(setq treesit-language-source-alist
      '((typescript "https://github.com/tree-sitter/tree-sitter-typescript" "master" "typescript/src")
        (tsx "https://github.com/tree-sitter/tree-sitter-typescript" "master" "tsx/src")))

(treesit-install-language-grammar 'typescript)
(treesit-install-language-grammar 'tsx)
```

2. Open a file and run M-x tsx-ts-mode
3. Then start eglot M-x eglot
4. Type <div classNa
5. M-x completion-at-point

Current behavior:

Complete with single quotes to <div className=''

Expected behaviour:

Complete with double quote to <div className=""

Rational & Observation:

In TSX it's common to use double quotes as it's more aligned with HTML style. Please also observe that you must have yasnippet installed to get completion.

**Comment by joaotavora** [comment]
Did you reach any conclusion?  I think I typed something to answer here but never pressed "send".  😄 so what I was going to say is that I think this is server-related: i.e. Eglot doesn't have a say in this matter of quotes.

**Reply by gormv** [reply]
Thanks! It just magically started to use " after a restart, so it solved itself :)
